.github/workflows/publish_device_startup.yml

```diff
@@ -24,8 +24,8 @@ jobs:
           docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
           docker buildx build \
             --platform linux/arm64 \
-            --tag huntsmanarray/camera_server:latest \
+            --tag huntsmanarray/device_startup:latest \
```

Comment:
`huntsmanarray/device`? This does more than just start each device, it also keep devices running.